SlidingUpPanel.js

```diff
@@ -115,6 +117,7 @@ class SlidingUpPanel extends React.PureComponent {
 
     const {top, bottom} = this.props.draggableRange
     const animatedValue = this.props.animatedValue.__getValue()
+    this._initialDragPosition = animatedValue
```

Comment:
```suggestion
    const initialValue = clamp(animatedValue, bottom, top)
    this._initialDragPosition = initialValue
```
Oh, somehow I missed this part. But it should be `initialValue` @quantosapplications.